open-apparel-registry/open-apparel-registry
# 1st pass of rebrand to Open Supply Hub

## Overview

Makes a few small tweaks to the site branding / logo for the beta period

Connects #1879 

## Demo

![localhost_6543_](https://user-images.githubusercontent.com/4432106/173126577-7f28a148-2cdb-44ef-b23a-d3ed296a0ded.png)
![localhost_6543_ (1)](https://user-images.githubusercontent.com/4432106/173126652-c851058a-ee01-420b-8d67-958099754d2f.png)


## Notes

I just made some guesses on how to style the login/logout/translation buttons in the navbar.

My assumption from "Make map greyscale" was that it applied to the facilities layer which had 4 blue colors (and not the basemap).
My first pass was to directly use the grayscale versions of the existing colors, but these were very hard to distinguish, so I settled on the current colors after some simple trial & error, but it's definitely quite basic.

I _could_ have made the primary button color on the entire site green to fulfill the requirements of "Changing the search button color", but based on the limited scope of the issue I left it at just the search button.

## Testing Instructions

* `scripts/server`
* Observe that the main map screen now has a green header with a grayscale map layer, and that the search button is also green

## Checklist
